core/src/test/scala/com/salesforce/op/stages/impl/feature/ToOccurTransformerTest.scala

```diff
@@ -57,9 +62,9 @@ class ToOccurTransformerTest extends FlatSpec with TestSparkContext {
   }
 
   lazy val (ds, leadId, numEmails, opptyId, doNotContact, numFormSubmits) =
-    TestFeatureBuilder("leadId", "numEmails", "opptyId", "doNotContact", "numFormSubmits", testData)
+    TestFeatureBuilder("leadId", "numEmails", "opptyId", "doNotContact", "numFormSubmits", extendedTestData)
 
-  Spec[ToOccurTransformer[_]] should "convert features to doolean using shortcuts" in {
+  Spec[ToOccurTransformer[_]] should "convert features to boolean using shortcuts" in {
```

Comment:
nit: you can do `it should` here (since the test name is already set in the base class)